client/app/templates/map.hbs

```diff
@@ -15,7 +15,7 @@
           {{fa-icon 'warning'}}
           <h3 class='msg__content' role='alert'>{{t 'map.messages.missingStudents'}}</h3>
         </div>
-        <i class='fa fa-close close' {{action 'closeAlert'}}></i>
+        <i class='fa fa-close close' onClick={{action 'closeAlert'}}></i>
```

Comment:
Ah, that's probably because the `i` element doesn't handle clicks by default. We should probably wrap this in a `button` that executes the action, since that would be more semantically correct anyway. Like...

```
<button {{action 'closeAlert'}}>
  <i class="fa fa-close close"></i>
</button>
```

This might require a bit more style legwork though.